Dockerfile

```diff
@@ -41,36 +41,33 @@ RUN apt-get clean \
   && apt-get update \
   && apt-get install -y \
   build-essential python3.9 python3-pip git cmake make ruby rake autoconf automake \
```

Comment:
Why do you install `python3.9`? Again, `docker build` might somehow swallow it, however, on `ubuntu:22.04`, only `python3` (which installs `python3.10`) is accepted:

```bash
# apt install python3.9
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package python3.9
E: Couldn't find any package by glob 'python3.9'
```

```bash
# apt install python3
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  libexpat1 libmpdec3 libpython3-stdlib libpython3.10-minimal libpython3.10-stdlib libreadline8 libsqlite3-0 media-types python3-minimal python3.10 python3.10-minimal readline-common
Suggested packages:
  python3-doc python3-tk python3-venv python3.10-venv python3.10-doc binutils binfmt-support readline-doc
The following NEW packages will be installed:
  libexpat1 libmpdec3 libpython3-stdlib libpython3.10-minimal libpython3.10-stdlib libreadline8 libsqlite3-0 media-types python3 python3-minimal python3.10 python3.10-minimal readline-common
0 upgraded, 13 newly installed, 0 to remove and 4 not upgraded.
Need to get 6531 kB of archives.
After this operation, 23.5 MB of additional disk space will be used.
Do you want to continue? [Y/n] ^C
```

Reply:
I inherited this Dockerfile and have not been maintaining it as someone else was managing the dockerhub images. I was only migrating to the new conan 2 changes for the file.

That's why I'm open to any changes. I did not create this.

Reply:
Cool. :wink:

I think we should simply improve the image as much as possible.